packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -120,7 +120,7 @@ export class GroupHeaderBase extends React.Component<IGroupHeaderProps, IGroupHe
             onClick={this._onToggleCollapse}
             aria-expanded={group ? !group.isCollapsed : undefined}
             aria-controls={group && !group.isCollapsed ? groupedListId : undefined}
-            {...expandButtonProps}
```

Comment:
This will remove the ability for devs to override props via `expandButtonProps` entirely and is a regression.

Using `expandButtonProps` and a custom render function for `GroupHeader`, one can override the default `aria-label` which should address your accessibility issue in SharePoint. See:

https://github.com/OfficeDev/office-ui-fabric-react/blob/0ad274184555dc37d6c7b39c02e7803fa619abb5/packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.types.ts#L219

Reply:
An alternative approach that modifies the defaults is to make the following line conditional based on `group.isCollapsed`:

https://github.com/OfficeDev/office-ui-fabric-react/pull/9758/files#diff-7f3e4ebbda3ef743e6154ae2aeefce82L20

Reply:
Here's a Codepen showcasing the first approach (override by app developer via `groupProps`: https://codepen.io/kevintcoughlin/pen/KjJoyJ